cli/script/command-executor.ts

```diff
@@ -1644,3 +1648,77 @@ function getSdk(accessKey: string, headers: Headers, customServerUrl: string): A
 
   return sdk;
 }
+
+function createPatch(command: cli.ICreatePatchCommand): Promise<void> {
+  return Q.Promise<void>((resolve, reject) => {
+    const scriptPath = path.join(__dirname, "patch-scripts", "create-patch.sh");
```

Comment:
```suggestion
    const scriptPath = path.join(__dirname, "patch-scripts", "create-patch.sh");
```

Can we move these values inside the constants?

Reply:
https://github.com/dream-sports-labs/delivr-cli/commit/2b6ed541be5fe5660506f09c2d1e0f4f26e5c921